src/components/comment-form.js

```diff
@@ -63,20 +64,29 @@ const ResultForm = ({ user }) => {
     [user, dirty, lockAcquired]
   );
 
+  if (readOnly) {
+    return (
+      <div className='position-relative'>
+        {value}
+      </div>
+    );
```

Comment:
Relative position was used for status-notice, it is not necessary here.

```suggestion
    return <div>{value}</div>;
```

Reply:
removed